packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -67,9 +67,26 @@ export const FileViewer = ({
   name,
   onReloadCallback,
 }: FileViewerProps) => {
+  if (!url) return <ErrorComponent onError={onReloadCallback} />;
+
+  if (mimetype === 'application/pdf') return <PdfFileViewer url={url} />;
+
+  return (
+    <img
```

Comment:
Does it makes sense to have a test coverage for this for a different mime types as well? To check that we render what we actually want and pass

As for insipiration to discuss, but this for only one case:

```ts
  test('renders image tag when mime type is an image', () => {
    const { rerender } = render(<FileViewer url="test.jpg" mimetype="image/jpeg" name="test" onReloadCallback={errorCallback} />);
    
    expect(screen.getByRole('img')).toHaveAttribute('src', 'test.jpg');

    rerender(<FileViewer url="test.png" mimetype="image/png" name="test" onReloadCallback={errorCallback} />);
    expect(screen.getByRole('img')).toHaveAttribute('src', 'test.png');
  });
```

Reply:
@vkuprin, thank you, [add](https://github.com/team-monite/monite-sdk/pull/133/commits/9ce56a93218dff581e836c620fb529afa1a2d236). Do you have ideas on how we could test iframe embedding?

Reply:
I'm thinking of utilising our playwright setup